test/src/unit-ReadCellSlabIter.cc

```diff
@@ -242,15 +243,14 @@ TEST_CASE_METHOD(
   auto dom = array_->array_->array_schema_latest().domain();
 
   std::vector<tdb_shared_ptr<FragmentMetadata>> fragments;
-  tdb_shared_ptr<FragmentMetadata> fragment =
-      tdb::make_shared<FragmentMetadata>(
-          HERE(),
-          nullptr,
-          nullptr,
-          array_->array_->array_schema_latest_ptr(),
-          URI(),
-          std::make_pair<uint64_t, uint64_t>(0, 0),
-          true);
+  tdb_shared_ptr<FragmentMetadata> fragment = make_shared<FragmentMetadata>(
```

Comment:
As long as you're changing statements, you should also take the opportunity to change `tdb_shared_ptr` to `shared_ptr`. No need to be exhaustive about it.